index.js

```diff
@@ -850,7 +850,8 @@ const rpcMethods = {
             }
         }
         if (ps.TYPE === 'ENUM') {
-            payload.hm.ENUM = ps.VALUE_LIST[params[3]];
```

Comment:
I consider this a breaking change and i need the integer representation in val and the hm.ENUM property. This change would lead to some broken scripts and UIs in my Smarthome ;)

Reply:
Where did you get this line from? It's not included in the latest version anymore🤔

This is the line as of the latest commit, here it's a fully optional change in protocol:

https://github.com/dersimn/hm2mqtt.js/blob/4bd1205cf438e85212fc71acda1da4c4317cfd8d/index.js#L961

Reply:
It's from 8c07bd8586afb34b040982aba68d56b3c8735e70